egs/swbd/s5c/local/nnet3/run_raw_tdnn.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
typically these local/run*.sh script are 'complete', i.e. they contain all the required arguments to actually do something.